common-app-lib/src/cmr/common_app/static.clj

```diff
@@ -256,11 +256,15 @@
                                     (gdocs/all-generic-docs-toc options)
                                     read-generic-markdown-toc
                                     gdocs/format-toc-into-doc)
+                collection-umm-version (config/collection-umm-version)
+                collection-umm-version-snake (string/replace collection-umm-version "." "_")
                 generic-versions (md->html (gdocs/generic-document-versions->markdown))]
             {:status 200
              :headers {"Content-Type" "text/html; charset=utf-8"}
              :body (-> resource
                        slurp
+                       (string/replace "%COLLECTION-UMM-VERSION%" collection-umm-version)
+                       (string/replace "%COLLECTION_UMM_VERSION%" collection-umm-version-snake)
```

Comment:
I really thought we already had this variable. I added it because I'm tired of updating the version number in documentation to keep the API current. The thinking is that if people copy and paste examples into their script, you want them using the latest version we have.